src/kimmdy/topology/atomic.py

```diff
@@ -122,6 +122,56 @@ def from_top_line(cls, l: list[str]):
         )
 
 
+@dataclass()
+class Settle:
+    """Information about one settles
+
+    A class containing atom information as in the settle section of the topology.
+
+    From gromacs topology:
+    ; nr funct doh dhh
+    """
+
+    nr: str
+    funct: str
+    doh: str
+    dhh: str
+
+    @classmethod
+    def from_top_line(cls, l: list[str]):
+        return cls(
+            nr=l[0],
+            funct=l[1],
+            doh=l[2],
+            dhh=l[3],
+        )
+
+
+@dataclass()
+class Exclusion:
```

Comment:
This would allow for reworking the slow growth topology file generation exclusion part (https://github.com/hits-mbm-dev/kimmdy/blob/7daac56bbb8235a6b69f2e5f43bbf633c3014569/src/kimmdy/coordinates.py#L315) , right?

Reply:
yes, `settles` and `exclusions` are now also included in the data structure.

Reply:
Right now, `Exclusions` uses the same style of parameters as other atomic types, so I have `a..` `i` ,`j`, `k`, `l`. But in theory the maximum number of exclusions is not specified, so if we ever encounter a topology with rows of 5 atoms in the exclusions we'll have to add a parameter. It's unlikely though, so not worth making it inconsistent with the other atomics to accommodate for variable lengths.

Reply:
So, should I change that section in coordinates.py? To me, that would be part of this PR.

Reply:
I would also link exclusions, then